stl/inc/xstring

```diff
@@ -4274,9 +4264,6 @@ public:
         auto& _My_data    = _Mypair._Myval2;
         auto& _Right_data = _Right._Mypair._Myval2;
 
-        const bool _My_large    = _My_data._Large_mode_engaged();
-        const bool _Right_large = _Right_data._Large_mode_engaged();
-

```

Comment:
https://github.com/microsoft/STL/blob/60f18856c56389001c38a9929ef37bd5c01c4e47/stl/inc/xstring#L4272
Is it encouraged to move this `using _STD swap` into the if block where it is used? I'm not sure as find they are usually defined at the beginning of functions.